### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by JimboJoe**
## Problem
- *Global upgrade of the package*
- *Upgrade to latest version 2.9.5*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Kay0u
- [x] **Approval (LGTM)** : Kay0u
- [x] **Approval (LGTM)** : Maniack C
- **CI succeeded** : 
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/piwigo_ynh%20testing%20(Official)/)
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by maniackcrudelis** [comment]
Bump

**Comment by kay0u** [comment]
One more LGTM

File: scripts/remove [hunk 1]
```diff
@@ -12,48 +12,75 @@ source /usr/share/yunohost/helpers
...
+domain=$(ynh_app_setting_get --app=$app --key=domain)
+db_name=$(ynh_app_setting_get --app=$app --key=db_name)
+final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 
 #=================================================
 # STANDARD REMOVE
 #=================================================
 # REMOVE DEPENDENCIES
 #=================================================
+ynh_script_progression --message="Removing dependencies..."2
```
Comment (alexAubin) [inline scripts/remove]: Smol typo at the end here (is this the weight ?)

File: scripts/upgrade [hunk 2]
```diff
@@ -15,123 +12,154 @@ source /usr/share/yunohost/helpers
...
 
 #=================================================
-# SPECIFIC SETUP
+# SPECIFIC UPGRADE
 #=================================================
+# COPY FILES TO $FINAL_PATH
+#=================================================
+
+# sets extended pattern matching options in the bash shell
+shopt -s extglob
```
Comment (alexAubin) [inline scripts/upgrade]: That's a bit mysterious what this is used for but okay I guess :P

Comment (maniackcrudelis) [reply scripts/upgrade]: It's explained the line before ;)
It sets the extended pattern matching. That makes bash accept regex.

**Comment by alexAubin** [comment]
bump, sounds like it can be merged ?

**Comment by kay0u** [comment]
I don't want to take the responsibility of pressing the button before I say:

> Can be merged in 3 days

But you can do it if you want!

Otherwise:

Can be merged in 3 days

**Comment by alexAubin** [comment]
Well :D That PR is from March 17

So I iz clickin teh buton

**Pull request merged**